tslint.json

```diff
@@ -53,6 +53,7 @@
     "no-reference-import": true,
     "no-require-imports": true,
     "no-shadowed-variable": false,
+    "no-submodule-imports": [true, "three/examples/jsm"],
```

Comment:
Submodule imports are banned by default, so this adds threejs examples to the whitelist of exceptions.

Reply:
Hopefully, we won't need to use examples anymore